.github/workflows/test-disk-images.yml

```
@@ -20,4 +20,12 @@ jobs:
 
       - name: Run disk image download test
         run: |
-          python -u test_disk_image_downloads.py
\ No newline at end of file
+          python -u test_disk_image_downloads.py
+
+  workflow-keepalive:
+    if: github.event_name == 'schedule'
+    runs-on: ubuntu-latest
+    permissions:
+      actions: write
+    steps:
+      - uses: liskin/gh-workflow-keepalive@v1
```

```suggestion
  workflow-keepalive:
    if: github.event_name == 'schedule'
    runs-on: ubuntu-latest
    permissions:
      actions: write
    steps:
      - uses: liskin/gh-workflow-keepalive@f72ff1a1336129f29bf0166c0fd0ca6cf1bcb38c #v1
```

If the repo is hacked and v1 tag starts pointing to a harmful commit, we, at least, are not affected.